chapter5/src/distance_test.go

```diff
@@ -0,0 +1,125 @@
+package chapter5
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDistance_CalculateDistance(t *testing.T) {
+	oxford := CreateCity(51.45, 1.15)
+	turin := CreateCity(45.04, 7.42)
+	ml, km := CalculateHarvesineDistance(oxford, turin)
+	if ml < 0.0 || km < 0.0 {
+		t.Errorf("The value cant be less than zero")
+	}
```

Comment:
you can also use a boolean assert here, but it's good to see you try multiple approaches :-)